carma_novatel_driver_wrapper/launch/carma-novatel-driver-wrapper-launch.py

```diff
@@ -0,0 +1,48 @@
+
+#  Copyright (C) 2021 LEIDOS.
+
+#  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+#  use this file except in compliance with the License. You may obtain a copy of
+#  the License at
+
+#  http://www.apache.org/licenses/LICENSE-2.0
+
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations under
+#  the License.
+
+import launch
+import launch.actions
+from launch.actions.declare_launch_argument import DeclareLaunchArgument
+import launch_ros.actions
+from ament_index_python.packages import get_package_share_directory
+
+from launch import LaunchDescription
+from launch_ros.actions import Node
+
+from launch.actions import IncludeLaunchDescription
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+
+#Nodes
+carma_novatel_driver_wrapper = Node(
+    package = 'carma_novatel_driver_wrapper',
+    executable = 'carma_novatel_wrapper_node',
+    name='carma_novatel_driver_wrapper'
```

Comment:
Add a launch argument to set the log level, see the template for an example. Only difference, is for this node since its not being launched as a component you would do 
```
arguments=['--ros-args', '--log-level', log_level ]
```
instead of the extra_arguments line